docs/decisions/0002-no-longer-rely-on-portal-designer.rst

```diff
@@ -0,0 +1,71 @@
+
+Data Duplication for Enterprise Customer Configuration
+======================================================
+
+By configuring the Enterprise learner portal data through ``designer``, we inevitably introduced data duplication between ``designer`` and the Enterprise Customer configuration that is done through Django admin in the LMS (via ``edx-enterprise``).
+
+The Enterprise Customer configuration done in Django admin already includes the majority of data the Enterprise learner portal relies on: the Enterprise Customer's uuid, name, and logo image. The same data needs to be configured in the ``designer`` service for each Enterprise Customer, leading to the data duplication across ``designer`` and the LMS.
+
+********
+Decision
+********
+
+Due to the challenges outlined above, the Enterprise learner portal will no longer rely on the ``designer`` service. Instead, the configuration data for an Enterprise learner portal will be added to the Django admin page within the LMS (via ``edx-enterprise``):
+
+* Branding configuration for the banner
+    * Fields for the background and border HEX color values will be added to the ``EnterpriseCustomerBrandingConfiguration`` model.
+* Learning coordinator email address
+    * A field for this data will be added to the ``EnterpriseCustomer`` model.
+* A field will be added to the ``EnterpriseCustomer`` model to specify which Enterprise Customers have the learner portal enabled.
+
+These changes within ``edx-enterprise`` will remove our data dependency on ``designer`` and allow us to instead fetch the necessary data for the Enterprise learner portal through existing API endpoint(s) in ``edx-enterprise``.
+
+************
+Consequences
+************
+
+The decision to no longer rely on the ``designer`` service to fetch the configuration data about an Enterprise learner portal has the following consequences:
+
+1. Engineers and other internal teams (e.g., ECS) that configure the metadata for Enterprise Customers will no need to configure data in two separate places.
+2. There will no longer be data duplication between the Enterprise Customer metadata defined in the LMS (via ``edx-enterprise``) and ``designer``. Instead, the configured data in the LMS will become the "source of truth" for the Enterprise learner portal.
+3. Configuration for the Enterprise learner portal will no longer be tightly coupled to the configuration for the Programs learner portal.
+4. The Enterprise engineering team will transition ownership of ``designer`` to the Programs team.
+5. A tech debt ticket will be added to the Enterprise engineering backlog to remove any models/views/etc. related to Enterprise from the ``designer`` service (ENT-2553).
+
```

Comment:
Not sure where this should go, but I think another reason we moved away from portal-designer was the lack of defined requirements for a full-blown CMS. We may consider publishing custom pages for enterprises down the line (at which point we may start to use portal-designer again, possibly), but we still need a baseline learner portal experience without a ton of customization, and this is the experience we are focused on delivering for the foreseeable future.